build/config.yaml

```diff
@@ -2,6 +2,12 @@
 version: 1.0
 sidecars:
   default:
+    X_CSI_SPEC_VERSION: v0.2
```

Comment:
Should we default to v1.0 and the latest sidecar versions?  That way it will work on k8s 1.15 without any changes, right?

Reply:
Yes, it would, but on the other side it would break older deployments if this is not set? Not sure which direction is better...